.env.example

```
@@ -43,3 +43,8 @@ OPENAI_API_KEY=your-openai-api-key
 MCP_LOG_LEVEL=INFO
 MCP_LOG_FILE=mcp.log
 MCP_DISABLE_CONSOLE_LOGGING=false
+
+# ─────────────────────────────────────────────────────────────────────────────
+# Falkor database
+# ─────────────────────────────────────────────────────────────────────────────
+FALKORDB_PASSWORD=change-me-since-this-is-fake
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)